build/handlers/index.js

```diff
@@ -26,12 +26,12 @@ function defaultPage(path) {
   ]
 }
 
-function fetchParseFix(resource) {
+function fetchParseFix(resource, processResultFn = processResult) {
```

Comment:
Passing this in as a default. This way I can mock it in the tests. Since I can't really run `build:fetch` without the keys, I was hoping @webmasterkai or @kevee could have a second look at this. I don't think it will cause any problems.

Reply:
We actually don't do the API build in CircleCI but instead [download the data](https://github.com/COVID19Tracking/website/blob/master/.circleci/config.yml#L21) using `npm run setup`.

Reply:
Ah, okay. I'm a little confused about our CI and build processes then. Where and when do the steps in `npm run build` get run? I kind of assumed CircleCI did testing and deployment.

Reply:
Oh, I think I get it now. I did not look closely enough at the CircleCI config. I assumed it was doing the entire `npm run build`. So, the `build:fetch` and `build:data-archive` steps are still being done elsewhere.